pkg/client/client.go

```diff
@@ -120,6 +121,14 @@ func (o *OptimizelyClient) IsFeatureEnabled(featureKey string, userContext entit
 		o.logger.Info(fmt.Sprintf(`Feature "%s" is not enabled for user "%s".`, featureKey, userContext.ID))
 	}
 
+	if o.notificationCenter != nil {
+		decisionNotification := decision.FeatureNotification(featureKey, &featureDecision, &userContext)
+
+		if e := o.notificationCenter.Send(notification.Decision, *decisionNotification); e != nil {
+			o.logger.Warning("Problem with sending notification")
```

Comment:
Should this be an error-level log, instead of warning?

Reply:
this is what it was before. Actually I was thinking to remove at all, as we have registry injection for this, and I cannot think of the scenario where it will become an error

Reply:
I think warning is fine as this isn't severe enough (catastrophic) to be an error